b3lb/rest/models.py

```diff
@@ -400,6 +400,12 @@ class Meta(object):
     def __str__(self):
         return "{}-{}".format(self.tenant.slug, str(self.sub_id).zfill(3))
 
+    @property
+    def get_valid_secret(self) -> str:
+        if self.secret2:
```

Comment:
This is fragile and may break key rollover. It requires to use the secret that was used to create the meeting, didn't it?